restli-int-test/src/test/java/com/linkedin/restli/examples/TestTyperefCustomDoubleAssociationKeyResource.java

```diff
@@ -69,13 +68,15 @@
   @BeforeClass
   public void initClass() throws Exception
   {
+    System.setProperty(RestLiIntTestServer.VALIDATE_KEYS_PROPERTY_NAME, Boolean.TRUE.toString());
```

Comment:
System properties are shared. Setting it this way might cause flakiness if tests are running in parallel.

You should ideally pass these as parameter to the init method. (There are similar methods already). To make this generic, you can add a init method that takes in RestLiConfig directly.